### ISSUE TITLE
Nonexistent signal after adding StaticBody to a MeshInstance from editor

### ISSUE THREAD

**Issue by lupoDharkael** [godotengine/godot#20521]
**Godot version:**
3.1.dev.custom_build.bfb21be

**OS/device including version:**
Fedora 28: Linux 4.17.7-200.fc28.x86_64
Mesa 18.0.5
VGA compatible controller: Intel Corporation 3rd Gen Core processor Graphics Controller
Kernel driver in use: i915
VGA compatible controller: NVIDIA Corporation GF117M [GeForce 610M/710M/810M/820M / GT 620M/625M/630M/720M] (rev a1)
Subsystem: Acer Incorporated [ALI] GeForce 710M
Kernel driver in use: nouveau

**Issue description:**
After launching the project, the Debugger shows the following warning
```
0:00:00:0287 - Nonexistent signal: changed
----------
Type:Error
Description: Nonexistent signal: changed
Time: 0:00:00:0287
C Error: Condition ' !s ' is true.
C Source: core/object.cpp:1465
C Function: disconnect
```
I shows one for every MeshInstance with StaticBody and collisions created from the editor (Convex or Trimesh).
![imagen](https://user-images.githubusercontent.com/14951430/43335970-8236a438-91d0-11e8-86d8-5bb15a893eca.png)

**Steps to reproduce:**
1. Create new project.
2. Add Spatial base node for the scene.
3. Add a MeshInstance with a CubeMesh, select the mesh Instance and add collisions from the editor (Create Convex Static Body).
4. Run it.

**Comment by Zireael07** [comment]
"master" isn't enough to be useful - which commit hash?

**Comment by lupoDharkael** [comment]
I've added the --version

**Comment by sketchyfun** [comment]
I'm glad I found this, I was going mad trying to locate the offending signal/connection in my projec,

**Comment by Noshyaar** [comment]
Closing as Fixed by #20381
